src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  background-color: white;
+}
+
+* {
+  margin: 0 auto;
+  width: 300px;
+  height: 100px;
+}
+
+div:hover {
+  height: 200px;
+}
+
+#header {
+  background-color: blue;
+}
+
+#footer {
+  background-color: black;
+}
+
+div.important {
```

Comment:
do not add unnecessary div, try to make selectors as simple as possible

Reply:
done